solidity/contracts/converter/BancorConverter.sol

```diff
@@ -765,23 +765,18 @@ contract BancorConverter is IBancorConverter, SmartTokenController, ContractRegi
         // destroy _amount from the caller's balance in the smart token
         token.destroy(msg.sender, _amount);
 
-        // iterate through the reserve tokens and send a percentage equal to the ratio between _amount
-        // and the total supply from each reserve balance to the caller
-        IERC20Token reserveToken;
-        uint256 reserveBalance;
-        uint256 reserveAmount;
+        // iterate through the reserve tokens and send a percentage equal to the ratio between
+        // _amount and the total supply from each reserve balance to the caller
         for (uint16 i = 0; i < reserveTokens.length; i++) {
-            reserveToken = reserveTokens[i];
-            reserveBalance = reserveToken.balanceOf(this);
-            reserveAmount = formula.calculateLiquidateReturn(supply, reserveBalance, totalReserveRatio, _amount);
-
-            Reserve storage reserve = reserves[reserveToken];
+            IERC20Token reserveToken = reserveTokens[i];

```

Comment:
same gas costs as before?
(I guess it's less critical in fund/liquidate)

Reply:
@yudilevi:
All the changes in `fund` and `liquidate` (and generally in this branch) are pretty cosmetic.
For some reason, GitHub's diff viewer doesn't show them properly here.
But if you compare it manually (or via some other diff tool), then you'll see that the only "real" difference in each one of these functions, is that I took `reserves[reserveToken]` and placed it directly where it is being used, since it is being used only in one single place (hence no need for the local variable).

Reply:
yeah I actually see it here but generally speaking, isn't it cleaner to define vars just once instead of each time in the loop?

Reply:
@yudilevi : It's a matter of opinion. In my opinion, it's cleaner to have fewer lines of code. And with the each var being declared and initialized in the same line, we get fewer lines of code. The only reason to declare them outside the loop is if we use them outside the loop.
BTW, with the current compiler, even that is not necessary. But this is due to the compiler bug that we spoke about earlier, so we may as well write our code correctly (under the assumption that this bug will be fixed in future compiler versions).

Reply:
sounds good